context/app/static/js/components/detailPage/provenance/ProvTable/ProvTable.tsx

```diff
@@ -92,6 +92,7 @@ function ProvEntityColumnContent({
                 uuid={item.uuid}
                 id={item.hubmap_id}
                 entity_type={item.entity_type}
+                ariaLabelText={`Tile representing ${item.entity_type} ${item.uuid}`}
```

Comment:
I wonder if a `useEntityTileAriaLabel` hook could help? That would allow us to implement more complex logic depending on the content type (dataset/sample/donor).

Similar concern as below with using UUID vs. HBM ID in aria labels.